awsauth_test.go

```diff
@@ -732,7 +746,7 @@ func TestAWSGetCredentials_shouldBeENV(t *testing.T) {
 
 // invalidAwsEnv establishes a httptest server to simulate behaviour
 // when endpoint doesn't respond as expected
-func invalidAwsEnv(t *testing.T) func() {
+func invalidAwsEnv() func() {
 	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		w.WriteHeader(400)
```

Comment:
just a general linter q: noticed this line wasn't caught by the linter like in `mock.go`, is this expected? thought maybe it was b/c it's embedded within another function or in general test cases are ignored?

Reply:
It looks like https://github.com/tommy-muehle/go-mnd ignores test files by default. 👍 I'm not sure if there's a way to enable that.

Reply:
ahh i see makes sense, the config object is hardcoded